src/main/java/pixelitor/utils/Messages.java

```diff
@@ -103,15 +104,19 @@ public static void showFilesSavedMessage(int numFiles, File dir) {
      */
     public static void showInStatusBar(String msg) {
         assert !msg.startsWith("<html>");
-        msgHandler.showInStatusBar("<html>" + msg);
+        if (msgHandler instanceof GUIMessageHandler guiMessageHandler) {
+            guiMessageHandler.showInStatusBar("<html>" + msg);
+        }
     }
 
     /**
      * Shows a non-HTML text message in the status bar.
      */
     public static void showPlainInStatusBar(String msg) {
         assert !msg.startsWith("<html>");
-        msgHandler.showInStatusBar(msg);
+        if (msgHandler instanceof GUIMessageHandler guiMessageHandler) {
```

Comment:
The whole purpose of the Messages/MessageHandler/GUIMessageHandler/TestMessageHandler classes is to separate the GUI code from the non-GUI code, and this change starts to mix them again.